Makefile.am

```diff
@@ -37,6 +37,7 @@ SUBDIRS += tests
 
 VENV_DIR = $(HOME)/.venv/vdsm
 VENV = $(VENV_DIR)/bin
+RPM_TOPDIR = $(PWD)/build
```

Comment:
Do we really use the current working directory?  Or do we want to use the vdsm directory: $(lastword $(MAKEFILE_LIST))?  I'd prefer the latter, it's safer and IMO generally better.

Reply:
Sounds better, yes. I took the same strategy as in `imageio`, but I agree with you that it would be safer. I'll try your proposal and change it.

Reply:
@mz-pdm: What is the vdsm directory? Why using it is better?

Reply:
Do you mean running "make -C /path/to/vdsm/dir"? Can you show an example creating build
directory in the wrong place with this pr?

Reply:
> Do you mean running "make -C /path/to/vdsm/dir"?

I was thinking of `make -f /path/to/vdsm/dir`. But this probably wouldn't work anyway and `-C` should be used instead.  OK.

Reply:
I tried for completeness. With -f fails due to the dependencies with other files. Fails to find them since they are linked with path relative to the repository root. If you invoke the script from somewhere else, they just break.

With -C works fine, it actually changes the directory before invoking, so it creates the `build` folder in the excepted place in the repository root.

I read the manual for other options and tried to think how they could break this. I could hack it by adding `-I` to include directories for dependencies on a run with `-f`. But nobody will realistically do that, ever. So I will keep it as it is for now, as it is kept consistent across projects and is secure enough. But it was a good point to bring up and test!